File: TestPlatform/src/Services/TestManager/VietGeeks.TestPlatform.TestManager.Core/ReadonlyModels/AggregatedGrading.cs

using System.Collections.Generic;
using VietGeeks.TestPlatform.TestManager.Core.Models;

namespace VietGeeks.TestPlatform.TestManager.Core.ReadonlyModels;
public class AggregatedGrading
{
    public GradingCriteriaConfigType GradingType { get; set; }
    public Dictionary<string, string>? Grades { get; set; }
    public PassMarkGrade? PassMarkGrade {get;set;}
}

public class PassMarkGrade
{
    public bool? IsPass { get; set; }
    public decimal? FinalPoints { get; set; }
    public decimal? TotalPoints { get; set; }
    public decimal? PassValue {get;set;}
    public RangeUnit Unit { get; set; }
}

File: TestPlatform/src/Services/TestManager/VietGeeks.TestPlatform.TestManager.WebApp/ClientApp/apps/test-runner/src/app/api/models/aggregated-grading.ts
/* tslint:disable */
/* eslint-disable */
import { PassMarkGrade } from './pass-mark-grade';
export interface AggregatedGrading {
  grades?: null | {
[key: string]: string;
};
  gradingType?: number;
  passMarkGrade?: PassMarkGrade;
}
